docs/contributing/ui_guidelines.md

```diff
@@ -0,0 +1,47 @@
+# UI Guidelines
+
+Wagtail’s user interface is built with:
+
+- **HTML** using [Django templates](https://docs.djangoproject.com/en/stable/ref/templates/language/).
+- **CSS** using [Sass](https://sass-lang.com/) and [Tailwind](https://tailwindcss.com/).
```

Comment:
Is it worth mentioning BEM here? Or is that really up for discussion now that we're all in with Tailwind?

Reply:
Ignore this. We have this clarified below.